Tests/RegressionTests.cs

```diff
@@ -29,7 +29,7 @@ public void BasicTemplateAlgorithm()
                 {"Total Trades", "1"},
                 {"Average Win", "0%"},
                 {"Average Loss", "0%"},
-                {"Compounding Annual Return", "311.484%"},
+                {"Compounding Annual Return", "280.522%"},
```

Comment:
I ran the tests on the latest master branch using the json and default data files in Github. The only test still failing for me is the UniverseSelectionRegressionAlgorithm (the only one I didn't change), but this is because the data files required for it are missing.